packages/valory/skills/hello_world_abci/handlers.py

```diff
@@ -19,15 +19,29 @@
 
 """This module contains the handler for the Hello World skill."""
 
-from packages.valory.skills.abstract_round_abci.handlers import ABCIRoundHandler
+from packages.valory.skills.abstract_round_abci.handlers import (
+    ABCIRoundHandler as BaseABCIRoundHandler,
+)
+from packages.valory.skills.abstract_round_abci.handlers import (
+    ContractApiHandler as BaseContractApiHandler,
+)
 from packages.valory.skills.abstract_round_abci.handlers import (
     HttpHandler as BaseHttpHandler,
 )
+from packages.valory.skills.abstract_round_abci.handlers import (
+    LedgerApiHandler as BaseLedgerApiHandler,
+)
 from packages.valory.skills.abstract_round_abci.handlers import (
     SigningHandler as BaseSigningHandler,
 )
+from packages.valory.skills.abstract_round_abci.handlers import (
+    TendermintHandler as BaseTendermintHandler,
+)
 
 
```

Comment:
All handlers so we have parity with the other skills

Reply:
What about dialogues?

Reply:
Forgot, done in https://github.com/valory-xyz/open-autonomy/pull/1696/commits/9c1d93ab454fefac6f5dc07e0b85c10b825ba8e5